includes/ComingSoon.php

```diff
@@ -86,31 +87,37 @@ public function notice_display() {
 		}
 	}
 
-
 	/**
-	 * Customize the admin bar.
+	 * Customize the admin bar with site status.
 	 *
 	 * @param \WP_Admin_Bar $admin_bar An instance of the WP_Admin_Bar class.
 	 */
-	public function add_tool_bar_item( \WP_Admin_Bar $admin_bar ) {
+	public function newfold_site_status( \WP_Admin_Bar $admin_bar ) {
 		if ( current_user_can( 'manage_options' ) ) {
-			$allowed_adminbar_html = array(
-				// div with inline styles
-				'div' => array(
-					'style' => array()
+			$is_coming_soon   = 'true' === get_option( 'nfd_coming_soon', 'false' );
+			$status           = $is_coming_soon
+			? '<span id="nfd-site-status-text" style="color:#E01C1C;">' . esc_html__( 'Coming Soon', 'newfold-module-coming-soon' ) . '</span>'
```

Comment:
We may want to output both statuses and just show/hide them with CSS in our JS method so we don't have to reload the page to toggle.

Reply:
Yea, I like that approach. Should make the toggle a bit easier if it doesn't need to know about the content. I'll get that in here before merging.

Reply:
This is in place here now.